app/views/bulkrax/exporters/_form.html.erb

```diff
@@ -60,6 +60,23 @@
     hint: 'leave blank or 0 for all records',
     label: t('bulkrax.exporter.labels.limit') %>
 
+  <%= form.input :date_filter,
+                 as: :boolean,
+                 label: t('bulkrax.exporter.labels.filter_by_date') %>
+  <div id="date_filter_picker" class="hidden">
+    <%= form.input :start_date,
+                   as: :date,
+                   label: t('bulkrax.exporter.labels.start_date') %>
+
+    <%= form.input :finish_date,
+                   as: :date,
+                   label: t('bulkrax.exporter.labels.finish_date') %>
+  </div>
+  <%= form.input :work_visibility,
+                 collection: form.object.work_status_list,
+                 label: t('bulkrax.exporter.labels.status') %>
```

Comment:
I think these labels should have i18n values in the [bulkrax.en.yml](https://github.com/samvera-labs/bulkrax/blob/export-from-date-range/config/locales/bulkrax.en.yml) file

Reply:
Do we need them? I'll add them, but I have never added them on projects in the past, unless I needed a label that said something different. I18n will auto-generate a human readable label based on the text after the last '.' which is all we usually need for english.

Reply:
I've noticed that sometimes, if a translation is missing, a hover will be added to the element that says "missing translation:...". I think this depends on the element because it doesn't always happen. 

Either way, I feel like it would be a future-proof change as we do eventually want to [support all the same languages that Hyrax does](https://github.com/samvera-labs/bulkrax/issues/255).

Reply:
Especially if we want to move out of samvera-labs into core samvera.

Reply:
fair enough, changes have been pushed.